The patch for usr/www/avm/html/de/fon_config/fon_config_1.js removes FON 3

--- usr/www/avm/html/de/fon_config/fon_config_1.js	2011-06-15 13:23:29.000000000 +0200
+++ usr/www/avm/html/de/fon_config/fon_config_1.js	2011-08-14 20:57:19.713641051 +0200
@@ -125,7 +125,6 @@
 if(firstNotation==-1)
 firstNotation = 2;
 g_NameList[2] = new String(name);
-jslAddOption("uiViewPort", "2", g_txtPort3, false);
 }
 <? if neq $var:DeviceType 'Tam' `
 for ( var i = 1; i < 9; i++) {
